test/node.js

```diff
@@ -0,0 +1,46 @@
+import test from 'ava';
+import pEvent from 'p-event';
+import path from 'path';
+import execa from '..';
+
+process.env.PATH = path.join(__dirname, 'fixtures') + path.delimiter + process.env.PATH;
+
+test('node()', async t => {
+	const {exitCode} = await execa.node('test/fixtures/noop');
+	t.is(exitCode, 0);
+});
+
+test('node pipe stdout', async t => {
+	const {stdout} = await execa.node('test/fixtures/noop', ['foo'], {
+		stdout: 'pipe'
+	});
+
+	t.is(stdout, 'foo');
+});
+
+test('node correctly use execPath', async t => {
```

Comment:
This is not correct use of those options. They should only be used to set a different Node.js executable. And `exec.node(..)` should only execute Node.js compatible scripts.

Reply:
Is there a way to check that the `execPath` is a valid Node.js executable ? There's so much difference between Node.js version/fork that I doubt we can find any common ground.
If the user bend `.node()` to run something else than Node.js it doesn't do any harm ?
In any case, I don't see any way to have a different Node.js executable path on CI.

Reply:
I guess it's fine to keep it like this for testing, as long as we don't document or recommend using it like this.